docs/Permissions/Local-Permissioning.md

```diff
@@ -1,15 +1,9 @@
-description: Pantheon Permissions feature
+description: Local Permissions 
 <!--- END of page meta data -->
 
-# Permissions 
+# Local Permissions 
 
-A permissioned network is a network where only specified nodes and accounts (participants) can participate. 
-Nodes and accounts outside those specified are prevented from participating. Permissioned networks can have node permissions enabled, 
-account permissions enabled, or both. 
-
-![Node Permissions](../images/node-permissioning-bad-actor.png)
-
-![Account Permissions](../images/account-permissioning.png)
+Local permissions supports node and account whitelisting. 
```

Comment:
Are you deliberately using "permissions" instead of "permissioning"?

Reply:
How about "Locally configured permissions can be used for both node and account whitelisting."

Reply:
It's more words but I am having trouble with "permissions supports"

Reply:
Updated to permissioning